src/libraries/System.Private.CoreLib/src/System/Char.cs

```diff
@@ -532,7 +615,7 @@ public static bool IsDigit(string s, int index)
             char c = s[index];
             if (IsLatin1(c))
             {
-                return IsInRange(c, '0', '9');
+                return IsBetween(c, '0', '9');
```

Comment:
Consider replacing all of these `IsBetween(c, '0', '9')`s with `IsAsciiDigit(c)`.

I'll accept any sort of "no", from "I'm too lazy" to "it's important to not overwhelm the poor, overworked JIT inlining evaluator in these routines"

Reply:
I was concerned about "the poor, overworked JIT" part of it and thus left them untouched.  If Levi says I should change them, though, I'm happy to.

Reply:
I have no thoughts one way or another. Worry about the JIT is in the hands of more knowledgeable people. 🙂